### PR TITLE
luci-app-advanced-reboot & luci-app-vpnbypass: fix uci require for ma…

### PR DESCRIPTION
**Pull request by stangri**
…ster

Signed-off-by: Stan Grishin <stangri@melmac.net>

@hnyman -- as per your request, this is the emergency fix for using uci modules in luci apps in master. I only have an ssh access to my dev machine, I cannot create a PR for the one-line fix (without other changes to the code/localisations) in a reasonable time remotely.

### CODE REVIEW

**Comment by hnyman** [comment]
I merged that.

I think that you have a fix also for vpn-policy-routing app. Is that needed also?
(Should it be cherry-picked for your repo or are you making a PR?)

File: applications/luci-app-vpnbypass/luasrc/model/cbi/vpnbypass.lua [hunk 1]
```diff
@@ -1,5 +1,7 @@
 readmeURL = "https://github.com/openwrt/packages/blob/master/net/vpnbypass/files/README.md"
 
+uci = require "uci"
```
Comment (jow-) [inline applications/luci-app-vpnbypass/luasrc/model/cbi/vpnbypass.lua]: This does not appear to be needed

**Comment by stangri** [comment]
@hnyman @jow -- I have no way of testing either change for the next few days. Please advise.

**Comment by hnyman** [comment]
>  I have no way of testing either change for the next few days. Please advise.

@stangri 
Just check the commit log. Jow already fixed things yesterday.

https://github.com/openwrt/luci/commit/75ac400168da8feb7ae193d5ef0e419bcaac8aa5
https://github.com/openwrt/luci/commit/1104b837cdf87e42512a32c48c9bedbd9c1d6d57

**Pull request merged**